ui/app/src/Router.tsx

```diff
@@ -12,14 +12,15 @@
 // limitations under the License.
 
 import { Routes, Route } from 'react-router-dom';
-import App from './App';
 import Docs from './views/Docs';
+import ViewDashboard from './views/ViewDashboard';
 
 function Router() {
   return (
     <Routes>
-      <Route path="/" element={<App />} />
+      <Route path="/projects/:projectID/dashboards/:dashboardID" element={<ViewDashboard />} />
       <Route path="docs" element={<Docs />} />
+      <Route path="/" element={<ViewDashboard />} />
```

Comment:
Just curious, what do you think _should_ be on the home page for Perses?

Reply:
I think it will be interesting to have the list of dashboards as a home page (for a first version).
I'm planning to create a simple view of the dashboard list a bit after this PR

Perhaps we should do something like Grafana and have a custom home page where you can pin some dashboards